--- gdome2-0.8.1/gtk-doc/Makefile.am.orig	2002-04-04 08:58:04.000000000 +0200
+++ gdome2-0.8.1/gtk-doc/Makefile.am	2014-01-18 08:52:44.335300888 +0100
@@ -33,9 +33,9 @@
 	rm -rf sgml html gdome2-decl-list.txt gdome2-decl.txt 
 
 install-data-local:
-	-install -d -m 0755 $(TARGET_DIR)
-	-install -m 0644 $(srcdir)/html/*.html $(TARGET_DIR)
-	-install -m 0644 $(srcdir)/html/index.sgml $(TARGET_DIR)
+	-install -d -m 0755 $(DESTDIR)$(TARGET_DIR)
+	-install -m 0644 $(srcdir)/html/*.html $(DESTDIR)$(TARGET_DIR)
+	-install -m 0644 $(srcdir)/html/index.sgml $(DESTDIR)$(TARGET_DIR)
 	-(cd $(DESTDIR); gtkdoc-fixxref --module=$(DOC_MODULE) --html-dir=$(HTML_DIR))
 
 .PHONY : html sgml templates scan
